config.yml

```
@@ -0,0 +1,50 @@
+---
+variables:
+  PRODUCT: &PRODUCT 90427
+  BASE_URL: &BASE_URL /path/to/product/build/doc/    # CHANGE THIS
+
+project: MLX{product}
+
+json_folder: [*BASE_URL, *PRODUCT, /database.json]
+# URL: [*BASE_URL, *PRODUCT, /rom/html/document.html#id]
+URL: [*BASE_URL, *PRODUCT, /html/document.html#id]
```

```suggestion
html_dir: [*BASE_URL, *PRODUCT, html]
```
`document.html#id` is unnecessary in the configuration. `node_clicked` in forcegraph.js can append the document name and node ID to `html_dir`.